frontend/ui/widget/bookmapwidget.lua

```diff
@@ -78,6 +78,18 @@ function BookMapRow:getPageX(page, right_edge)
     return x
 end
 
+function BookMapRow:getIndicatorXY(page, shift_y)
```

Comment:
Fine with factorizing this.
`shift_y` (which reads like it accepts a number by which it will be shifted) should be named `shift_down` if a boolean is to be provided.